Firestore/core/src/remote/datastore.h

```diff
@@ -212,6 +216,7 @@ class Datastore : public std::enable_shared_from_this<Datastore> {
   GrpcConnection grpc_connection_;
 
   std::vector<std::unique_ptr<GrpcCall>> active_calls_;
+  core::DatabaseInfo database_info_;
```

Comment:
save database_info as class attribute to read datastore.databaseId out of it later.